SDC/newrelic.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: [''],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '',
```

Comment:
It's good that you don't have an API key in here, but where is it? This doesn't look like it's coming from a .env file or anything, so how is your New Relic integration working?